implementations/rust/ockam/ockam_command/src/util/api.rs

```diff
@@ -238,18 +238,16 @@ fn node_service(service_name: &str) -> String {
 pub mod enroll {
     use ockam_api::cloud::enroll::auth0::{Auth0Token, AuthenticateAuth0Token};
 
-    use crate::enroll::*;
-
     use super::*;
 
-    pub(crate) fn auth0<'a>(
-        cmd: EnrollCommand,
+    pub fn auth0<'a>(
+        route: &MultiAddr,
```

Comment:
❤️ we should make this type of change across the board - so UI layer objects `*Command` aren't injected into underlying logic.